atlas-spec/mpc-engine/src/lib.rs

```diff
@@ -56,6 +62,7 @@ pub const COMPUTATIONAL_SECURITY: usize = 128 / 8;
 /// The statistical security parameter, in bytes.
 pub const STATISTICAL_SECURITY: usize = 128 / 8;
 
+pub mod broadcast;
```

Comment:
Can you add some comment on what's part of the spec (model) and what is needed to make it usable/executable?

Reply:
Right, I will clarify that this is a utility module in lieu of a specification of proper peer-to-peer broadcast.